src/Raven.Server/Documents/Queries/Results/TimeSeriesRetriever.cs

```diff
@@ -202,11 +199,7 @@ BlittableJsonReaderObject GetAggregatedValues()
 
                 _argumentValuesDictionary?.Clear();
 
-                return _context.ReadObject(new DynamicJsonValue
-                {
-                    ["Count"] = count,
```

Comment:
Can we use `nameof` for the "Count" and "Results"?

Reply:
done